### PR TITLE
Added changes to support TWO-WAY Authentication while connecting to oracle DB while bringing up JRF domain

### PR DESCRIPTION
**Pull request by ganeshs05**
Current implementation was not supporting TWO-WAY SSL while connecting to RCU schema on Oracle SSL DB.
Added changes to set properties to support the same while configuring WebLogic Domain.

### CODE REVIEW

**Review by CarolynRountree — APPROVED** [review]

**Review by jshum2479 — COMMENTED** [review]
Where is the documentation update on this changes ?   You should update 

documentation/2.0/content/rcuinfo.md

**Comment by ganeshs05** [comment]
> Where is the documentation update on this changes ? You should update
> 
> documentation/2.0/content/rcuinfo.md

Documentation was already present for all trust/keystore  properties...only properties required for TWO_WAY SSL DB communication was not set while creating weblogic domain

File: core/src/main/python/wlsdeploy/tool/create/domain_creator.py [hunk 1]
```diff
@@ -966,6 +967,32 @@ def __set_atp_connection_property(self, root_location, property_name, property_v
         self.wlst_helper.set(wlst_name, wlst_value)
 
         root_location.remove_name_token(property_name)
+    
+    def __set_atp_connection_property_encrypted(self, root_location, property_name, property_value):
```
[the remark this answers is not in the document]
Comment (ganeshs05) [reply core/src/main/python/wlsdeploy/tool/create/domain_creator.py]: Corresponding changes has been done and  pushed please verify

**Review by robertpatrick — CHANGES_REQUESTED** [review]
If you are adding functionality, you must update the documentation--either in this or a separate PR.  We will consider this PR once we see the documentation changes.

**Review by jshum2479 — APPROVED** [review]

File: documentation/2.0/content/rcuinfo.md [hunk 2]
```diff
@@ -64,9 +64,26 @@ domainInfo:
       oracle.net.tns_admin: <absolute path of the unzipped wallet root directory>
 
 ```
+
+For an Oracle SSL database with ONE_WAY SSL enabled, with an `SSO` wallet, use the following example:
+```yaml
+domainInfo:
+    RCUDbInfo:
+      useSSL : true
+      rcu_db_conn_string: <reuired URL string for use with -run_rcu>
```
Comment (rakillen) [inline documentation/2.0/content/rcuinfo.md:73]: *required - this is spelled "reuired" in 4 places, including old and new examples

Comment (ganeshs05) [reply documentation/2.0/content/rcuinfo.md]: updated, thanks for the review

**Review by rakillen — APPROVED** [review]

**Review by robertpatrick — APPROVED** [review]

**Pull request merged**